interface/src/app/gene/gene-search-form.tpl.html

```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="col-xs-12">
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <div class="col-xs-3">
+          <h4>Search for Genes</h4>
+          <form class="form">
+            <textarea name="genes" rows="7" ng-model="genesToAdd.query"
+              placeholder="Search genes by symbol or Entrez ID">
+            </textarea>
+            <span class="error">{{ errors.name }}</span>
+            <br>
+            <button class="btn btn-primary" ng-click="searchGenes()"
+              ng-disabled="loadingSearchResults">
+              Search   <i class="fa fa-search"></i>
```

Comment:
Yes, thanks for catching that - having multiple spaces before the icon was just a typo - it'll be fixed in the next commit.
Yeah, for the indenting, that is what I chose to go with - I looked but didn't see anything in the Google style guide to discourage breaking the tag line like that (and the required indenting is 2 spaces).